ext/bg/js/anki-note-builder.js

```diff
@@ -126,6 +120,7 @@ class AnkiNoteBuilder {
         if (reading) { filename += `_${reading}`; }
         if (expression) { filename += `_${expression}`; }
         filename += '.mp3';
+        filename = filename.replace(/\]/g, '');
```

Comment:
Since you added it, are there other characters that have to be escaped or sanitized from the filename? If it's actually stored in the filesystem under that name, then `/` and `\` can't be used either, but there are also other characters that are disallowed in filenames.

Reply:
The reason why "]" is removed is due to how Anki handles them, not necessarily file name restrictions. That is probably a valid thing to do however.

https://github.com/ankitects/anki/blob/913c26870529174b50f4207397ede7444a48dc6e/pylib/anki/utils.py#L94
https://github.com/ankitects/anki/blob/913c26870529174b50f4207397ede7444a48dc6e/pylib/anki/exporting.py#L67
https://github.com/ankitects/anki/blob/913c26870529174b50f4207397ede7444a48dc6e/pylib/anki/media.py#L35

Reply:
Yeah, makes sense that it would remove the `]` character since it's used to terminate the `[audio:...]` block.
(offtopic, Anki seems to ignore empty `[audio:]` audio blocks in those functions)

Reply:
> are there other characters that have to be escaped or sanitized from the filename?

(Just FYI, I consider this out of scope for this change, but will gladly make the update in a follow-up PR.)

Reply:
Thanks for clarifying, I was waiting for a new commit addressing this issue